--- pytest-6.2.2/testing/acceptance_test.py~	2021-01-25 15:51:22.000000000 +0100
+++ pytest-6.2.2/testing/acceptance_test.py	2021-02-28 15:25:46.086814937 +0100
@@ -142,6 +142,7 @@ class TestGeneralUsage:
             return (DummyDist(entry_points),)
 
         monkeypatch.setattr(importlib_metadata, "distributions", my_dists)
+        monkeypatch.delenv("PYTEST_DISABLE_PLUGIN_AUTOLOAD", raising=False)
         params = ("-p", "mycov") if load_cov_early else ()
         pytester.runpytest_inprocess(*params)
         if load_cov_early:
